test/sagemaker_tests/tensorflow/tensorflow2_training/resources/mnist/mnist_custom.py

```diff
@@ -117,8 +117,8 @@ def load_data(data_dir):
     print(vx.shape, vy.shape)
     # x.shape = (1000, 784), y.shape = (1000, )
 
-    x, y = x.astype(np.float32), y.astype(np.int)
-    vx, vy = vx.astype(np.float32), vy.astype(np.int)
+    x, y = x.astype(np.float32), y.astype(int)
```

Comment:
Numpy np.int is deprecated and causing assertion errors, np.int is logically equivalent and is an alias to int in python. [See numpy doc for more info](https://numpy.org/devdocs/release/1.20.0-notes.html#using-the-aliases-of-builtin-types-like-np-int-is-deprecated)